src/node/types.h

```diff
@@ -39,6 +41,13 @@ struct BlockCreateOptions {
      * larger CompactSize encoded lengths.
      */
     size_t block_reserved_weight{8000};
+
+    /*
+     * This is a sane minimum value for block_reserved_weight.
+     * Reserving weight for coinbase transaction and block header
```

Comment:
I would leave out this sentence, or say "Setting a lower value is prevented at startup."

Reply:
Done